en/orm/validation.rst

```diff
@@ -474,6 +476,46 @@ You may want to conditionally apply rules based on entity data::
         return false;
     }, 'userExists');
 
+Conditional/Dynamic Error Messages
+----------------------------------
+
+Rules, being it :ref:`custom callables <creating-a-rules-checker>`, or
```

Comment:
This section could be a link instead of duplicate content.

Reply:
It's quite similar, but still a little bit different, rule objects vs validator objects, the rule syntax in general, the requirement to set the `errorField` option. I could imagine that some people would get things mixed up when just being referred to the validation section :worried:

Reply:
ok